### DISCUSSION TITLE
March | Community Manager Monthly Report - Jesse(JDubs)

### DISCUSSION THREAD

**Discussion by 2xJDubs** [ringdao-community-guild/discussion#124] [Show and tell]
# Overview

March was productive in that we push forwarded on a number of initiatives that will benefit the community.  I have discussed a few other initiatives with the the Ambassadors and David that I will be introduce this month.

These initiatives include the following:
- Launching the DCDAO vote around DAO staking with InvArch.
- Helped David launch the farming LP rewards proposal. 
- Helped move forward the plan for Distributing Darwinia's $PINK allocation.

New Initiatives:
- Proposal for Treasury support in active members in governance until staked RING can be used for voting.
- Working with June our DCDAO Advocate on some additional community engagement things.
- Cross-community airdrop with Avail.
- Minor restructuring of the Ambassador program.

# Details
## Community Engagement

**Quiz Games**
Worked with the Ambassadors to run Quiz Games # 15, # 16, # 117 and # 18 were run on Discord with a typical 20 or so members joining and one week seeing 30+.

**Weekly News**
Working with June, our DCDAO Advocate, to collate weekly news that is then posted on the DCDAO Twitter account. 

**Monthly DCDAO Update**
Working with June and David on a monthly medium article summarizing what happened with DCDAO the previous month.

#Community Management

- Continued to engage and help our community members with staking and migration.
- Published the proposal for the Q2 budget for Community Management.
- Continued to support Helix and their Discord.
- Working with David to grow the DCDAO Advocates.
- Meet with Kyle from Avail to discuss supporting our respective communities and engaging them.

# April Plan

I expect to be doing the following:

- DCDAO to host Quizzes each week
- Continued engagement with the Community
- Continue Weekly News
- Continue to develop the Advocate program and recruit more Advocates with David's help
- Work on adding 1 new Ambassador
- Continued management of the Ambassador and Advocate programs
- Propose a minor restructuring of the Ambassador program
- Work with InvArch to setup the DAO staking
- Submit the HydraDX Omnipool listing proposal and move that effort forward
- Work on the other initiatives mentioned in the Overview above.

# Funds Wallet

Please remit salary to 0xebd9A48eD1128375EB4383ED4d53478B4FD85a8D

**Comment by hackfisher** [comment]
> Proposal for Treasury support in active members in governance until staked RING can be used for voting.

This is in progress and can be partially resolved after runtime upgrade which will apply the RFC of https://github.com/orgs/darwinia-network/discussions/1455, the staking RINGs can be unstaked immediately to participate in Gov.

After that, we can practice on Tally after the recent [successful integration with it](https://github.com/orgs/dcdao/discussions/87#discussioncomment-8985000), we will probably adopt if for our next version of Gov Design. 

Ref: https://github.com/orgs/darwinia-network/discussions/1284

**Reply by 2xJDubs** [reply]
Will The Tally integration allow for RING that is staked or in deposit to be used in voting and what is the timeline for supporting this?  Personally I would not want to unbond my stake to vote and lose out on rewards but I also shouldn't be penalized in not being able to vote using my staked RING as I could provide greater support for initiatives if I could vote with my staked and deposited RING.  What I am thinking of proposing would be to delegate RING to active voters so they could participate more in governance until we can vote with staked/deposited RING.  If the Tally integration will solve this in less than 30 days I agree there is no need for my proposal but if it will be 60 to 90 days then it may be a worthwhile experiment.

**Reply by hackfisher** [reply]
Update: https://github.com/orgs/darwinia-network/discussions/1481
